test/cli_wait_test.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
+
+	"github.com/go-check/check"
+	"github.com/gotestyourself/gotestyourself/icmd"
+)
+
+// PouchWaitSuite is the test suite for wait CLI.
+type PouchWaitSuite struct{}
+
+func init() {
+	check.Suite(&PouchWaitSuite{})
+}
+
+// SetUpSuite does common setup in the beginning of each test suite.
+func (suite *PouchWaitSuite) SetUpSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+
+	environment.PruneAllContainers(apiClient)
+
+	PullImage(c, busyboxImage)
+}
+
+// TearDownTest does cleanup work in the end of each test.
+func (suite *PouchWaitSuite) TearDownTest(c *check.C) {
+}
+
+// TestWaitNonBlockedExitZero is to verify the correctness of waiting a non-blocking container with 0 exit code
+func (suite *PouchWaitSuite) TestWaitNonBlockedExitZero(c *check.C) {
+	name := "TestWaitNonBlockedExitZero"
+	command.PouchRun("run", "-d", "--name", name, busyboxImage, "sh", "-c", "true").Assert(c, icmd.Success)
+	defer DelContainerForceMultyTime(c, name)
+
+	output := command.PouchRun("wait", name).Stdout()
```

Comment:
I would prefer change this code to:
```
res := command.PouchRun("wait", name)
res. Assert(c, icmd.Success)
output := res.Stdout()
```
In this way, we could get more info if `pouch wait` failed.